docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/AnomalyDetection/RandomizedPcaSample.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.AnomalyDetection
+{
+    public static class RandomizedPcaSample
+    {
+        /// <summary>
+        /// Example with 3 feature values.
+        /// </summary>
+        private class DataPoint
+        {
+            [VectorType(3)]
+            public float[] Features;
+        }
+
+        /// <summary>
+        /// Class used to capture prediction of <see cref="DataPoint"/> in <see cref="Example"/>.
+        /// </summary>
+        // We disable this warning because complier doesn't realize those fields below are assigned somewhere.
+#pragma warning disable 649
```

Comment:
can we disable this somewhere else? csproj etc?

Reply:
Yes, we can do
```git 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <OutputType>Exe</OutputType>
+    <NoWarn>1701;1702;0649</NoWarn>
   </PropertyGroup>
```

---
In reply to: [261047307](https://github.com/dotnet/machinelearning/pull/2780#discussion_r261047307) [](ancestors = 261047307)